### ISSUE TITLE
Play Scene/Camera not working in Unity 2021.2.2f1

### ISSUE THREAD

**Issue by Voltecite** [Kodrin/URP-PSX#3]
Whenever you load one of the scenes to test it out it's working in the Scene editor perfectly but when switching over to Play Mode it doesn't seem to load anything, only showing a black screen.

Disabling the cameras Post processing allows the camera to function normally but without the effects.

Possible fix:
Go into the file "Assets/UniversalRenderPipelineAsset.asset" in Unity.

In the Inspector select the "Depth Texture" option to enable it.

After enabling the Depth Texture option it seems to be working like normal with no issues.

I am unaware if it was supposed to be enabled by default but through my experience it was not.

For further assistance I put a reference photo of the option below.
![Screenshot 2021-11-18 163250](https://user-images.githubusercontent.com/86991748/142507419-82d68599-a8d1-47cf-9fb7-9cf19f82e661.png)
.

**Comment by TylerCode** [comment]
I can't offer a ton of insight into why this is happening but I can confirm that URP 12 and Unity 2021.2 break some things. I was going through some other documentation from last year and none of it worked for fullscreen shaders in 2021.2

The related repo is here. I noticed there was an additional Editor script if it helps anything. https://github.com/Cyanilux/URP_BlitRenderFeature

**Comment by Wopsie** [comment]
I was able to get everything working just fine after ticking the Depth Texture box. Im on version 2021.2.15f1. Something to note however when implementing in your own project is to enable post-processing on your camera. Furthermore some shaders have some .cginc includes, so you may need to adjust those paths in the shader code if you want to use a different folder structure

**Comment by Alvarden15** [comment]
Yeah, i check said shaders, and some of them directly refer to specific folders that, if moved, won't work correctly. So far, those are the shaders i found whose folder directions has to be changed in order to be moved anywhere and work correctly:
- Fog.shader (referrs to Assets/Shaders/cginc/voronoi.cginc)

**Comment by Alvarden15** [comment]
For those reading this, there is a more updated fork of this asset that is compatible with 2021 out of the box, plus a new CRT visual effect:
https://github.com/Math-Man/URP-PSX-FORKED
